akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -335,7 +332,7 @@ private[remote] class Association(
       if (log.isDebugEnabled) {
         val reason =
           if (removed) "removed unused quarantined association"
-          else s"overflow of send queue, size [$queueSize]"
+          else s"overflow of send queue, size [$qSize]"
```

Comment:
Is `qSize == queueSize`?

Reply:
This is not obvious to me, either

Reply:
@jrudolph @raboof Indeed, this GitHub diff doesn't explain it fully. 
[Here is the value it was referencing originally](https://github.com/akka/akka/blob/ae5172cf16f5e6082e9eb2ed164fabf0c44a8ec3/akka-remote/src/main/scala/akka/remote/artery/Association.scala#L158)
```scala private val queueSize = advancedSettings.OutboundMessageQueueSize ```

[Here is the outer function](https://github.com/akka/akka/blob/ae5172cf16f5e6082e9eb2ed164fabf0c44a8ec3/akka-remote/src/main/scala/akka/remote/artery/Association.scala#L318-L342)

[Specifically, the inner function the change made is in](https://github.com/akka/akka/blob/ae5172cf16f5e6082e9eb2ed164fabf0c44a8ec3/akka-remote/src/main/scala/akka/remote/artery/Association.scala#L326-L335)

The `$qSize` had a warning unused which made me look at it. Because it was only using the outer `$queueSize` and not the `$qSize` passed into the function it was in, I made the change. Otherwise why is it there?

Reply:
Agreed, this seems like a valid change that is actually a bug fix. I think as it is the error message is a less helpful that it could be because it doesn't specify whether the regular queue or the control queue overflowed, so maybe you could create a ticket to revisit that?